source/administration/devops-command-center.rst

```diff
@@ -82,21 +93,35 @@ You can use the test commands to create incidents that are populated with random
 Playbooks and Incidents
 -----------------------
 
-Incidents and playbooks are associated with teams in Mattermost. Incident channels are created based on playbooks, which define whether an incident channel is public or private. Read more about `public and private channels <https://docs.mattermost.com/help/getting-started/organizing-conversations.html>`_.
+Playbooks and incidents are associated with teams in Mattermost. Incident channels are created based on playbooks, which define whether an incident channel is public or private. Read more about `public and private channels <https://docs.mattermost.com/help/getting-started/organizing-conversations.html>`_.
 
 Only members of the team in which the playbook or incident is defined have access. Playbook membership is independent of incident membership.
 
 - Members of a playbook may start an incident using that playbook, and edit the playbook's stages and steps.
 - Members of an incident may modify the current state of the incident, and invite new members to the incident channel.
 
-During an active incident, you want to focus on triaging and solving the problem as soon as possible. Planning your incident support strategy ahead of time with playbooks is the best way to ensure incidents run smoothly. A playbook is a recipe for dealing with and resolving an incident. In a playbook, you can plan ahead so that during an incident responders know exactly what to do. Make sure to schedule a retrospective analysis to iterate on the design of your playbooks after the incident finishes.
+During an active incident, you want to focus on triaging and solving the problem as soon as possible. Planning your incident support strategy ahead of time with playbooks is the best way to ensure incidents run smoothly. A playbook is a recipe for dealing with and resolving an incident. In a playbook, you can plan ahead so that during an incident responders know exactly what to do.
 
-Within each playbook, you can create stages and tasks to ensure that items are addressed and completed in sequential order. The tasks can optionally be associated with slash commands and assigned to individual team members.
+Within each playbook, you can create stages and tasks to ensure that items are addressed and completed in sequential order. The tasks can optionally be associated with slash commands and assigned to individual team members. Playbooks also include status update settings, including a reminder feature, so regular updates can be shared to selected channels at set intervals. Make sure to schedule a retrospective analysis to iterate on the design of your playbooks after the incident finishes.
 
 Once complete, incident channels can be exported using the channel export option for analysis. Teams can identify bottlenecks in the incident by seeing time gaps between when checklist items are completed and incorporating necessary changes into the playbook for next incident.
 
+Incident status updates
```

Comment:
@lieut-data, I've added this here because it doesn't fall squarely under just playbooks or just incidents. I'm 0/5 on where it lives though, so feedback welcome.